testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -124,6 +124,26 @@ impl EmitJobRequest {
             },
         }
     }
+
+    pub fn fixed_tps(instances: Vec<Instance>, tps: u64) -> Self {
+        let mut worker_num: usize = 10;
+        while instances.len() * worker_num > tps as usize {
+            if worker_num == 1 {
+                break;
+            }
+            worker_num -= 1;
+        }
+        let wait_time = (instances.len() * worker_num * 1000_usize / tps as usize) as u64;
```

Comment:
Do you mind extracting out function like
fn fixed_tps_params(instance_count: usize, tps: u64) -> (wait_time, num_workers)

And adding few unit tests on this fn demonstrating how it behaves?